array-manipulation-1/src/compact.js

```diff
@@ -0,0 +1,14 @@
+/* exported compact */
+function compact(array) {
+  var compactArray = [];
+  for (var i = 0; i < array.length; i++) {
+    if (array[i]) {
+      compactArray.push(array[i]);
+    }
+    if (!array[i]) {
```

Comment:
you will notice if you remove this that the function will still work, because it won't do anything if the value is not truthy